_topic_map.yml

```diff
@@ -1759,8 +1759,8 @@ Topics:
   File: understanding-sandboxed-containers
 - Name: Deploying OpenShift sandboxed containers workloads
   File: deploying-sandboxed-container-workloads
-- Name: Disabling OpenShift sandboxed container workloads
-  File: disabling-sandboxed-container-workloads
+- Name: Uninstalling OpenShift sandboxed containers workloads
```

Comment:
"containers workloads" makes me twitch, but I see some precedent on line 1760.  I'll find some way to console myself.

Reply:
Thanks for the suggestion. 

The approved product name is OpenShift sandboxed containers so I can't really change from "containers" to "container" :-)